soc/nordic/nrf54h/bicr/bicrgen-schema.json

```diff
@@ -0,0 +1,422 @@
+{
+  "title": "nRF54H20 BICR Configuration",
+  "type": "object",
+  "properties": {
+    "power": {
+      "type": "object",
+      "title": "Power supply configuration",
+      "properties": {
+        "scheme": {
+          "type": "string",
+          "title": "Power supply scheme",
+          "enumNames": [
+            "Unconfigured, system will not boot",
+            "VDDH supplied with 2.1-5.5V and VDD regulated by the chip, inductor present",
+            "Both VDD and VDDH supplied with 1.8V, inductor present"
+          ],
+          "enum": [
+            "UNCONFIGURED",
+            "VDD_VDDH_1V8",
+            "VDDH_2V1_5V5"
+          ],
+          "default": "UNCONFIGURED"
+        }
+      },
+      "required": [
+        "scheme"
+      ]
+    },
+    "ioPortPower": {
+      "type": "object",
+      "title": "IO port power configuration",
+      "properties": {
+        "p1Supply": {
+          "type": "string",
+          "title": "P1 power supply (VDDIO_P1)",
+          "enumNames": [
+            "Not supplied (P1 not used)",
+            "VDDIO_P1 shorted to VDD",
+            "VDDIO_P1 connected to an external 1.8V supply or to VDD_EXT"
```

Comment:
Suggest to make "VDDIO_P1 connected to an external 1.8V supply or to VDD_EXT" the first option after "Not supplied" because it is the best option when it can be used. "SHORTED" is a bit hard to explain, and it will cause the device to not enter low power modes when VDDH_2V1_5V5 is selected. This is for all ports.

Reply:
done